src/vm/amd64/unixasmhelpers.S

```diff
@@ -304,7 +232,7 @@ NESTED_ENTRY OnHijackInteriorPointerTripThread, _TEXT, NoHandler
 
     END_PROLOGUE
 
-    call                C_FUNC(OnHijackInteriorPointerWorker)
+    call                C_FUNC(OnHijackWorker)
 
     free_stack          0x8
     pop_register        rax
```

Comment:
@jkotas: I've implemented your recommendation in this commit: https://github.com/dotnet/coreclr/pull/6522/commits/f10be428c72a2d22fd59443797cdde59102dc9ea